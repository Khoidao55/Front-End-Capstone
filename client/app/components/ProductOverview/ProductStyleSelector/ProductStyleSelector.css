.styles-individual-thumbnail {
  width: 10%;
}