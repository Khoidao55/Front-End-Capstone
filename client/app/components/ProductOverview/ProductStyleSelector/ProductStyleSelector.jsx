import React from 'react';

const ProductStyleSelector = ({ handleSelectedStyleClick, styles }) => {
  if (styles !== undefined) {
    return (
      <div className="styles-thumbnail">
        <h2>Styles > {styles[0].name}</h2>
        {styles.map((productStyle) => {
          return(
          <div className="styles-thumbnail-list">
            <img onClick={handleSelectedStyleClick} value={productStyle} style={{borderRadius: '50%', width: '50px', height: '50px'}} src={productStyle.photos[0].thumbnail_url} />
          </div>
          )
        })}
      </div>
    );
  }

  return (
    <div />
  );
};
export default ProductStyleSelector;
