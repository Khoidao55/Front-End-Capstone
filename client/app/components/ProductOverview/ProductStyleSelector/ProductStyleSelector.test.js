import React from 'react';
import { shallow } from 'enzyme';
import ProductStyleSelector from './ProductStyleSelector.jsx';

describe('Product Style Selector', () => {
  it('should render my component', () => {
    const wrapper = shallow(<ProductStyleSelector />);

    expect(wrapper).toMatchSnapshot();
  });
});
