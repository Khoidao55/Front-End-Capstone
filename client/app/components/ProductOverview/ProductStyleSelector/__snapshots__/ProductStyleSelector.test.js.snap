// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Style Selector should render my component 1`] = `ShallowWrapper {}`;
